README.md

```diff
@@ -1,110 +1,124 @@
-(WCAG)](https://www.w3.org/TR/wcag-3.0/).
+APCA was created by Andrew Somers at Myndex Research, and under the oversight of the by the Visual Contrast subgroup of Silver, and is the candidate method for contrast for [WCAG&nbsp;3](https://www.w3.org/TR/wcag-3.0/) (W3C Accessibility Guidelines).
 
-An interactive demo is available at <https://xi.github.io/apca-introduction/tool/>.
+An interactive demo is available at <https://www.myndex.com/APCA/>.
 
 ## Algorithm
 
 ```js
+
+      // Estimated screen Ys from simple RGB int array [255,255,255]
 function sRGBtoY(srgb) {
   var r = Math.pow(srgb[0] / 255, 2.4);
   var g = Math.pow(srgb[1] / 255, 2.4);
   var b = Math.pow(srgb[2] / 255, 2.4);
-  var y = 0.2126729 * r + 0.7151522 * g + 0.0721750 * b;
+  var ys = 0.2126729 * r + 0.7151522 * g + 0.0721750 * b;
 
-  if (y < 0.022) {
-    y += Math.pow(0.022 - y, 1.414);
+      // Soft black clamp for flare and other purposes.
+  if (ys < 0.022) {
+    ys += Math.pow(0.022 - ys, 1.414);
   }
-  return y;
+  return ys;
 }
 
-function contrast(fg, bg) {
-  var yfg = sRGBtoY(fg);
-  var ybg = sRGBtoY(bg);
-  var c = 1.14;
+    // Send text and bg color, as simple RGB int arrays [255,255,255]
+function contrastAPCA(tx, bg) {
+  var ystx = sRGBtoY(tx);
+  var ysbg = sRGBtoY(bg);
+  var sapc = 1.14;
+  var offset = 0.027;
 
-  if (ybg > yfg) {
-    c *= Math.pow(ybg, 0.56) - Math.pow(yfg, 0.57);
+      // Calculate raw SAPC contrast value
+  if (ysbg > ystx) {
+    sapc *= Math.pow(ysbg, 0.56) - Math.pow(ystx, 0.57);
   } else {
-    c *= Math.pow(ybg, 0.65) - Math.pow(yfg, 0.62);
+    sapc *= Math.pow(ysbg, 0.65) - Math.pow(ystx, 0.62);
   }
 
-  if (Math.abs(c) < 0.1) {
+      // Clamp low Lc values to 0, otherwise offset.
+  if (Math.abs(sapc) < 0.1) {
     return 0;
-  } else if (c > 0) {
-    c -= 0.027;
+  } else if (sapc > 0) {
+    sapc -= offset;
   } else {
-    c += 0.027;
+    sapc += offset;
   }
 
-  return c * 100;
+      // return APCA Lc value as a signed number
+  return sapc * 100;
 }
 ```
 
 ([Source](https://github.com/Myndex/SAPC-APCA/blob/master/documentation/APCA-W3-LaTeX.md))
 
-## Migrating from WCAG 2.x
+## Migrating from WCAG&nbsp;2.x
 
-- WCAG 2.x produces a ratio between 1:1 and 21:1. APCA produces a value roughly
-  between -100 and 100.
-- Unlike WCAG 2.x, APCA reports different values when you switch foreground and
-  background.
+- WCAG&nbsp;2.x produces a ratio between 1:1 and 21:1. APCA produces a value roughly
+  between -108 to 0 for light text (on a dark background), and 0 to 106 for dark text.
+- Unlike WCAG&nbsp;2.x, APCA reports different values when you switch text color and
+  background color.
 - The result of APCA is negative for light text on dark background. You will
   usually work with the absolute value though.
-- WCAG 2.x defined three thresholds: 3:1, 4.5:1, and 7:1. These roughly
+- WCAG 2.x defined three thresholds: 3:1, 4.5:1, and 7:1. These _roughly_
```

Comment:
better use `*` (emphasis) then `_` (underline)

Reply:
No, GitHub flavored markdown syntax is ` ** ` for bold and ` _ ` for italic.

Reply:
Ok, you are correct that `_` is not underline but italics in both [github flavored markdown](https://github.github.com/gfm/#emphasis-and-strong-emphasis) and [commonmark](https://spec.commonmark.org/0.30/#emphasis-and-strong-emphasis). But it's not bold/italic, its emphasis/strong emphasis. Bold/italic is only how this emphasis is expressed visually.

Anyway, in repo uses `*` rather than `_`.

Reply:
On GitHub, when editing, pressing command+I, or selecting the _I_ in the toolbar above, puts ` _ ` around the selected text. Pressing command+B or clicking the **B** on the toolbar above surrounds the selected text with ` ** `. If you have a problem with that I suppose you should take it up with GitHub.